src/django/api/tests.py

```diff
@@ -8496,11 +8496,11 @@ class ContributorManagerTest(TestCase):
     fixtures = ['users', 'contributors']
 
     def test_filter_by_name(self):
-        matches = Contributor.objects.filter_by_name('factory a')
+        matches = Contributor.objects.filter_by_name('facility a')
         self.assertGreater(matches.count(), 0)
 
         # No result should be returned for exact match
-        matches = Contributor.objects.filter_by_name('factory')
+        matches = Contributor.objects.filter_by_name('facility')
```

Comment:
Now that the changes to the contributors fixture have been simplified we need to revert this test change.

Reply:
Right, I forgot about this. I rebased on `ogr/develop` and removed the commit with these changes.